packages/core/src/types.ts

```diff
@@ -30,6 +30,7 @@ export interface ICatalogNode {
 export interface ICategory {
   name: string
   description: string
+  collapsed?: boolean
```

Comment:
Just a curiosity: why is there a ? behind the category name?

Reply:
The collapse property is [optional](https://www.typescriptlang.org/docs/handbook/2/objects.html#optional-properties). so when not given defaults to category being rendered expanded.